exercises/practice/zipper/lib/bin_tree.ex

```diff
@@ -18,7 +18,7 @@ defimpl Inspect, for: BinTree do
   # A custom inspect instance purely for the tests, this makes error messages
   # much more readable.
   #
-  # BinTree[value: 3, left: BinTree[value: 5, right: BinTree[value: 6]]] becomes (3:(5::(6::)):)
+  # %BinTree{value: 3, left: %BinTree{value: 5, right: %BinTree{value: 6}}} becomes (3:(5::(6::)):)
```

Comment:
Fixes #457

(I wasn't aware there is an issue for this, I found it by accident and decided to fix it right away)